infrastructure/images/k6-action/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM alpine:latest
```

Comment:
Better to depend on a actual version. Latest can have breaking changes without any notice, renovate will issue PRs for new versions so we should get them fairly quickly anyways

Reply:
Done!